### ISSUE TITLE
Paginated conditions issue

### ISSUE THREAD

**Issue by davscro** [TanvirArjel/EFCore.GenericRepository#29]
Hi,

it seems that Conditions for PaginationSpecification were not considered correctly.
If I have something like
```C#
var specification = new PaginationSpecification<Transaction>();
specification.Conditions.Add(transaction => transactionIds.Contains(transaction.Id));
specification.Conditions.Add(e => e.TransactionStatus == StatusEnum.ForSell);
specification.PageIndex = 1;
specification.PageSize = 20;
```
I got ```"totalItems": 1342,``` which is wrong. BUt if I do one liner condition like
```
specification.Conditions.Add(transaction => transactionIds.Contains(transaction.Id) && transaction.TransactionStatus == StatusEnum.ForSell);
```
everything is OK. Is there some bug there or I am missing something with PaginationSpecification?

**Comment by NotAsea** [comment]
this is bug in PaginatedList watch here #45

**Comment by TanvirArjel** [comment]
The issue has been fixed in versions 6.0.4 and 1.5.3!
